### PR TITLE
[15.0][MIG] maintenance_sign_oca: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @carolinafernandez-tecnativa and @pedrobaeza can you review it?

@Tecnativa TT41745

### CODE REVIEW

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM Thanks :)

**Comment by victoralmau** [comment]
Ping @pilarvargas-tecnativa

File: maintenance_sign_oca/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#. Go to Sign > Roles and create a new one with the following data:
```
Comment (pilarvargas-tecnativa) [inline maintenance_sign_oca/readme/CONFIGURE.rst]:
```suggestion
#. Go to Sign > Settings > Roles and create a new one with the following data:
```

**Review by pilarvargas-tecnativa — APPROVED** [review]
Functional review, thanks

**Comment by victoralmau** [comment]
Ping @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**